src/sentry/dynamic_sampling/rules/biases/recalibration_bias.py

```diff
@@ -17,7 +21,11 @@ class RecalibrationBias(Bias):
     """
 
     def generate_rules(self, project: Project, base_sample_rate: float) -> list[PolymorphicRule]:
-        adjusted_factor = get_adjusted_factor(project.organization.id)
+        if is_project_mode_sampling(project.organization):
+            adjusted_factor = get_adjusted_project_factor(project.id)
+        else:
```

Comment:
nit: should we explicitly check if it is organization sampling mode?

Reply:
Not in this case. This also applies to old organizations that were in neither mode.